### PR TITLE
update vscode readme

### PR DESCRIPTION
**Pull request by ppezaris**
**This PR Addresses:**  
[update vscode readme](https://trello.com/c/8JnRxvBS/5094-update-vscode-readme)

### CODE REVIEW

**Comment by ppezaris** [comment]
this is where

```
	milestone: {
		title: string;
		state: string;
		number: number;
		id: string;
		description: string;
	};
```
								
[shared/agent/src/protocol/agent.protocol.providers.ts](https://github.com/teamcodestream/codestream/blob/1c89f042f6db070f0ce63a25a4e40615badce054/shared/agent/src/protocol/agent.protocol.providers.ts#L468-L474) (Lines 468-474)

File: shared/agent/src/providers/github.ts [hunk 1]
```diff
@@ -1202,10 +1202,16 @@ export class GitHubProvider extends ThirdPartyIssueProviderBase<CSGitHubProvider
 	async getMyPullRequests(request: {
 		owner: string;
 		repo: string;
+		queries: string[];
 		isOpen?: boolean;
```
Comment (ppezaris) [inline shared/agent/src/providers/github.ts:1206]: commet here

File: shared/ui/store/providerPullRequests/actions.ts [hunk 2]
```diff
@@ -142,6 +142,8 @@ export const removeFromMyPullRequests = (providerId: string, id: string) =>
 
 export const getMyPullRequests = (
 	providerId: string,
+	queries: string[],
+	openReposOnly: boolean,
```
Comment (ppezaris) [inline shared/ui/store/providerPullRequests/actions.ts:146]: fix this thing

**Review by ppezaris — COMMENTED** [review]
look at

File: shared/agent/src/protocol/agent.protocol.providers.ts [hunk 3]
```diff
@@ -623,8 +623,7 @@ export const QueryThirdPartyRequestType = new RequestType<
 export interface GetMyPullRequestsRequest {
 	owner?: string;
 	name?: string;
-	/** if true, only return PRs that are open */
-	isOpen?: boolean;
+	queries: string[];
```
Comment (bcanzanella) [inline shared/agent/src/protocol/agent.protocol.providers.ts:626]: commenting on a PR!

**Comment by ppezaris** [comment]
do it here

```
		comments?: {
			totalCount: number;
		};
```
								
[shared/agent/src/protocol/agent.protocol.providers.ts](https://github.com/teamcodestream/codestream/blob/1c89f042f6db070f0ce63a25a4e40615badce054/shared/agent/src/protocol/agent.protocol.providers.ts#L454-L456) (Lines 454-456)

**Comment by ppezaris** [comment]
tenor
Screen Shot 2019-08-01 at 7 02 21 PM

File: shared/agent/src/providers/github.ts [hunk 4]
```diff
@@ -1218,19 +1224,18 @@ export class GitHubProvider extends ThirdPartyIssueProviderBase<CSGitHubProvider
...
-		let results: any = [];
 		const repoQuery =
 			request && request.owner && request.repo ? `repo:${request.owner}/${request.repo} ` : "";
-		const isOpenQuery = request && request.isOpen === true ? "is:open " : "";
 
-		const query = (q: string) => `query Search {
+		const queries = request.queries;
+		const buildQuery = (query: string) => `query Search {
 			rateLimit {
 				limit
```
Comment (ppezaris) [inline shared/agent/src/providers/github.ts:1233]: here are my thoughts

**Pull request closed without merging**